$OpenBSD: patch-source_glest_map_editor_CMakeLists_txt,v 1.3 2017/09/14 04:43:57 kirby Exp $
Program fails to start ("unable to init GTK") when help2man runs it.
Index: source/glest_map_editor/CMakeLists.txt
--- source/glest_map_editor/CMakeLists.txt.orig
+++ source/glest_map_editor/CMakeLists.txt
@@ -187,7 +187,7 @@ IF(BUILD_MEGAGLEST_MAP_EDITOR)
 	TARGET_LINK_LIBRARIES(${TARGET_NAME} libmegaglest)
 	TARGET_LINK_LIBRARIES(${TARGET_NAME} ${EXTERNAL_LIBS})
 
-	IF(EXISTS ${HELP2MAN} AND NOT APPLE)
+	IF(FALSE)
 		# on mac <> --help sometimes hangs, so we need "NOT APPLE" ^ or some timeout here for now
 		MESSAGE(STATUS "***-- Found help2man: ${HELP2MAN} about to build manpage in [${HELP2MAN_OUT_PATH}${TARGET_NAME_MANPAGE}]")
 
@@ -213,7 +213,7 @@ IF(BUILD_MEGAGLEST_MAP_EDITOR)
 		        ${TARGET_NAME} 
 		        DESTINATION ${MEGAGLEST_BIN_INSTALL_PATH})
 
-	        IF(EXISTS ${HELP2MAN} AND NOT "${MEGAGLEST_MANPAGE_INSTALL_PATH}" STREQUAL "" AND NOT APPLE)
+	        IF(FALSE)
          		MESSAGE(STATUS "**NOTE: Will Install manpage [${TARGET_NAME_MANPAGE}] to [${MEGAGLEST_MANPAGE_INSTALL_PATH}]")	
 
 		        # Installation of the program manpage file
